sklearn/datasets/base.py

```diff
@@ -272,8 +272,9 @@ def load_iris(return_X_y=False):
         Dictionary-like object, the interesting attributes are:
         'data', the data to learn, 'target', the classification labels,
         'target_names', the meaning of the labels, 'feature_names', the
-        meaning of the features, and 'DESCR', the
-        full description of the dataset.
+        meaning of the features, 'DESCR', the
+        full description of the dataset, 'filename' (added in version 0.19),
```

Comment:
I thought about it, but decided to go this way. At least for me it looks more straightforward and maintainable (for example if you decide to sort parameters by name in docstring later on).